docs/Proposals.md

```diff
@@ -0,0 +1,30 @@
+# Proposals
+
+Although Windranger aims to provide a modular framework for DAO governance, we will adopt and experiment
+with a few simple approaches.
+
+## Process #1
+
+1. Propose on forum
+2. If forum responses appear positive, create snapshot governance proposal
+3. If governance proposal passes, manually execute transactions contained in the proposal
+
+Advantages:
+
+- Provides mechanisms for community feedback before transactions are executed
+- Transactions are often explicitly included in a proposal, so exact actions can be easily verified
```

Comment:
In current approach you include tx data into proposal, you just vote on it onchain and then transaction executed via timelock with some delay and can be canceled (lets say by ByBit veto).

Reply:
There may be several paths. For example, a proposal may propose building a new voting strategy, which is not an on-chain component (belongs as part of snapshot, which is mostly off-chain).